src/pages/getting-started/helpdesk-apps/index.mdx

```diff
@@ -0,0 +1,158 @@
+
+<div style={{maxWidth: '450px'}}>
+  <img src="/images/getting-started/helpdesk-apps/livechat-helpdesk-apps-choosing-product.png" alt="Creating HelpDesk apps choosing the product" width="557px" height="386px"/>
+</div>
+
+If you do not have a HelpDesk account, you will be asked to activate it. If you want to activate HelpDesk and start a free trial, click **Activate HelpDesk** button.
+
+<div style={{maxWidth: '450px'}}>
+  <img src="/images/getting-started/helpdesk-apps/livechat-helpdesk-apps-activating account.png" alt="Creating HelpDesk apps activating account" width="279px" height="230px"/>
+</div>
+
+💡 **Need a helping hand?** Right after signing in to the Developer Console, you can launch the **App wizard**. We go there with guidelines for the LiveChat app creation process, but it can also be very helpful while exploring HelpDesk.
+
+### Step 2 Adding HelpDesk Widgets block
+
+You can form your app using components called Building blocks. Each building block works differently. To read more about Building Blocks, click the **Read more** redirect or go to the [App guides](/getting-started/app-guides) section in our documentation.
+
+In this guide, we will use the HelpDesk Widgets building block. Widgets are web apps that can interact and communicate with the HelpDesk App. They are loaded inside the HelpDesk App interface.
+
+<img
+  alt="Creating HelpDesk apps building blocks"
+  src="/images/getting-started/helpdesk-apps/livechat-helpdesk-apps-building-blocks.png"
+  width="1460px"
+  height="796px"
+/>
+
+To add a new Widget to your application, you need to follow these steps:
+
+1. Go to the **Building blocks** tab.
+2. Add a new HelpDesk Widgets block by clicking the **+ Add building block** button and then choosing **HelpDesk Widgets**.
+3. Enter the **Widget source URL**, which is the address of your application. If it's not ready yet, you can use our [sample apps](/extending-agent-app/agent-app-sdk#sample-apps) to play around.
+4. Now, go to the **Widget placement** and choose one of the options. It allows you to decide where exactly in the HelDesk App you want to display your integration. We've chosen the **Full screen** option in our example. In the HelpDesk App, Full screen apps are located in the navigation bar on the left, and they open maximized in the HelpDesk App. [Learn more...](/extending-agent-app#main-menu-fullscreen-apps)
+5. The **Opening state** section is not required and works only for the **Ticket Details** widget placement. It allows you to add components to your application and see a live preview of what the app will look like in the **Opening state preview**.
```

Comment:
```suggestion
5. The **Opening state** section isn't required and works only for the **Ticket Details** widget placement. It allows you to add components to your application and see a live preview of what the app will look like in the **Opening state preview**.
```

Reply:
"see a live preview of what the app will look like in the **Opening state preview**."
It doesn't really explain much

Reply:
I have a screen in the HelpDesk Widgets BB showing what it looks like, so I believe that if I link it here (like I described below about the component) that would be the best solution as for me, explaining it with words, might be confusing. This BB is created h[ere](https://github.com/livechat/livechat-public-docs/pull/1234)